test/test.py

```diff
@@ -21,19 +23,78 @@
 )
 
 
-print(e_plant)
-e_plant.tf()
-
+#print(e_plant)
+tf=e_plant.tf()
+#print(x)
+tf_hashed = hashlib.sha256(b"tf").hexdigest()
+#print(tf_hashed)
 
 controllers = e_plant.tune_controllers()
+_controllers = [c.toDict() for c in controllers ]
+#print(_controllers)
 sys_list = [
     system.ClosedLoop(
         controller = controller,
         plant = e_plant)
     for controller in controllers ]
 
 for system in sys_list:
-    system.step_response()
+    response=system.step_response()
+
+response_output = hashlib.sha256(b"response").hexdigest()
+
+#print(response_output)
+#List of controllers reference to verify the outputs
+controllers_list = [{'ctype': 'PID', 'Ms': '1.4', 'n_kp': 0.19147715177000726, 'n_ti': 0.5731720773427056, 'n_td': 1.6505422617593668, 'kp': 0.19147715177000726, 'ti': 0.6083527188887472, 'td': 1.751850643592542}, 
```

Comment:
Format list and dictionaries in multiple lines, to keep line short and readable, for example:
{
'key1': value,
'key2': value,
... etc
}